frontend/pages/hosts/ManageHostsPage/HostTableConfig.tsx

```diff
@@ -15,7 +15,7 @@ import StatusCell from "components/TableContainer/DataTable/StatusCell/StatusCel
 import TextCell from "components/TableContainer/DataTable/TextCell/TextCell";
 import {
   humanHostMemory,
-  humanHostUptime,
```

Comment:
I believe `humanHostUptime` can be deleted from `frontend/utilities/helpers.ts` now.